yb-voyager/src/srcdb/oracle.go

```diff
@@ -418,6 +437,10 @@ func (ora *Oracle) GetColumnToSequenceMap(tableList []sqlname.NameTuple) map[str
 			qualifiedColumnName := fmt.Sprintf("%s.%s", table.AsQualifiedCatalogName(), columnName)
 			columnToSequenceMap[qualifiedColumnName] = fmt.Sprintf("%s_%s_seq", tname, columnName)
 		}
+		err = rows.Close()
```

Comment:
If we have a defer function already for this, why do we need it explicitly? (please check all other places)

Reply:
We need it because we need to close rows object before the next iteration. Check this - https://github.com/yugabyte/yb-voyager/pull/1459#discussion_r1606332604

Defer will only run at the end of the function, so that is just a safety net in this case, really.

Reply:
Got it, thanks!